src/Aspire.Hosting.AWS.DynamoDBLocal/DynamoDBLocalResourceBuilderExtensions.cs

```diff
@@ -0,0 +1,163 @@
+                credentials = new BasicAWSCredentials("AKIAIOSFODNN7EXAMPLE", "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY");
+            }
+
+            var ddbClient = new AmazonDynamoDBClient(config);
+
+            // A problem with doing it at this point is there isn't access to get an 
+            // ILogger from this point to allow logging in my seed utilities.
+            var utilities = new SeedDynamoDBUtilities(ddbClient);
+
+            // TODO: This should eventually be added to a real lifecycle hookup that would have its own CancellationTokenSource
+            var cancellationTokenSource = new CancellationTokenSource();
+
+            _ = Task.Run(async () =>
+            {
+                utilities.WaitTillContainerAvailable(cancellationTokenSource.Token);
+
+                // TODO: Figure out to get an ILogger at this point so I can get logging without using Console.WriteLine
+                await seedDynamoDBCallback(utilities, cancellationTokenSource.Token).ConfigureAwait(false);
+            });
+        });
+
+        return builder;
+    }
+
+    /// <summary>
+    /// Add a reference to the DynamoDB local to the project. This is done by setting the AWS_ENDPOINT_URL_DYNAMODB environment
+    /// variable for the project to the http endpoint of the DynamoDB local container. Any DynamoDB service clients
+    /// created in the project relying on endpoint resolution will pick up this environment variable and use it.
+    /// </summary>
+    /// <typeparam name="TDestination"></typeparam>
+    /// <param name="builder"></param>
+    /// <param name="dynamoDBLocalresourceBuilder"></param>
+    /// <returns></returns>
+    /// <exception cref="DistributedApplicationException"></exception>
+    public static IResourceBuilder<TDestination> WithAWSDynamoDBLocalReference<TDestination>(this IResourceBuilder<TDestination> builder, IResourceBuilder<IDynamoDBLocalResource> dynamoDBLocalresourceBuilder)
+        where TDestination : IResourceWithEnvironment
+    {
+        builder.WithEnvironment(context =>
+        {
+            if (context.PublisherName == "manifest")
+            {
+                return;
+            }
+
+            if (!dynamoDBLocalresourceBuilder.Resource.TryGetAllocatedEndPoints(out var allocatedEndPoints))
+            {
+                throw new DistributedApplicationException("Expected allocated endpoints!");
+            }
+
+            var endpoint = allocatedEndPoints.Single();
+            var serviceUrl = $"http://{endpoint.Address}:{endpoint.Port}/";
+            context.EnvironmentVariables.Add("AWS_ENDPOINT_URL_DYNAMODB", serviceUrl);
+        });
+        return builder;
+    }
+
+    /// <summary>
+    /// Since Amazon DynamoDB local is a dev only resource skip writing it to the manifest
+    /// because it should not be part of a deployment.
+    /// </summary>
+    /// <param name="jsonWriter"></param>
+    private static void SkipWriteToManifest(Utf8JsonWriter jsonWriter)
```

Comment:
This can be removed since you won't need it if you use ```ManifestPublisherCallbackAnnotaiton.Ignore```.

Reply:
Removed